xbmc/AppParamParser.h

```diff
@@ -25,7 +27,7 @@ class CAppParamParser
 
   const CFileItemList& GetPlaylist() const;
 
-  int m_logLevel;
+  int m_logLevel = LOG_LEVEL_NORMAL;
```

Comment:
This default init required the new include of `commons/ilog.h`, to get the definition of `LOG_LEVEL_NORMAL`, right?

I would always prefer less include files in header files over default  member init to keep compile time dependencies as small as possible...

Here, I would default init with 0 and set the value `LOG_LEVEL_NORMAL` in the cpp file.

Reply:
removed this change